packages/calcite-components/src/components/navigation-logo/navigation-logo.tsx

```diff
@@ -59,6 +60,11 @@ export class CalciteNavigationLogo implements LoadableComponent {
   /** Specifies the `src` to an image. */
   @Prop() thumbnail: string;
 
+  /**
+   * Specifies the number at which section headings should start.
```

Comment:
@josercarcamo Very timely! @DitwanP made similar changes across the components earlier today. CC'ing him for editorial review on the `headingLevel` doc update.